packages/connect-webextension/src/index.ts

```diff
@@ -33,6 +33,12 @@ const logger = initLog('@trezor/connect-webextension');
 const popupManagerLogger = initLog('@trezor/connect-webextension/popupManager');
 let _popupManager: popup.PopupManager;
 
+const keepAlive = () => {
+    chrome.runtime.onMessage.addListener(() => {
+        return false;
+    });
+};
+
```

Comment:
wow, how this hack work? is there some sort of documentation that could be put into comment here?